Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/retention/ContactDiaryCleanTask.kt

```diff
@@ -29,6 +29,9 @@ class ContactDiaryCleanTask @Inject constructor(
         retentionCalculation.clearObsoleteContactDiaryPersonEncounters()
         Timber.tag(TAG).d("Obsolete contact diary person encounters cleaned up")
 
+        retentionCalculation.clearObsoleteRiskPerDate()
```

Comment:
The risk task will generate these data sets on it's own, independent of diary logic.
Does this task always run? Or could there be a case where this task doesn't run, because the user doesn't use the diary, and the "risk-per-date" datasets just accumulate in the databse?

Reply:
The task is planed in the Mainactivity and also triggered when not using the contact diary. Should be all right.